src/components/FileUpload/FileUploadTarget.js

```diff
@@ -42,9 +42,12 @@ const FileUploadTarget = props => {
           onChange={handleSelectedFile}
         />
         {error && error.graphQLErrors && (
-          <Header as="h4">
-            Failed to upload: {error.graphQLErrors.map(err => err.message)}
-          </Header>
+          <Message
+            negative
+            icon="warning circle"
+            header="Failed to upload:"
+            content={error.graphQLErrors.map(err => err.message)}
```

Comment:
Maybe `error.graphQLErrors.join(', ')` would be better so you have a formatted string instead of an array of strings.